site/src/pages/WorkspacesPage/batchActions.tsx

```diff
@@ -0,0 +1,76 @@
+import { useMutation } from "react-query";
+import {
+  deleteWorkspace,
+  startWorkspace,
+  stopWorkspace,
+  updateWorkspace,
+} from "api/api";
+import type { Workspace } from "api/typesGenerated";
+import { displayError } from "components/GlobalSnackbar/utils";
+
+interface UseBatchActionsProps {
+  onSuccess: () => Promise<void>;
+}
+
+export function useBatchActions(options: UseBatchActionsProps) {
```

Comment:
Hm..... looks like we have a custom here 👀 🕵️ 😆

Reply:
We'll take down Tanner Linsley someday

Reply:
you wrote this one too!!!

Reply:
lmao 😆